improver_tests/calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -1095,6 +1107,103 @@ def test_coefficients_truncnorm_realizations_no_statsmodels(self):
             [cube.name() for cube in result], self.expected_coeff_names
         )
 
+    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
+    def test_each_point(self):
+        """Test computing coefficients independently for each grid point (initial guess
+        and minimising) returns the expected coefficients and associated metadata."""
+        plugin = Plugin(self.distribution, each_point=True)
+        result = plugin.process(
+            self.historic_temperature_forecast_cube, self.temperature_truth_cube
+        )
+        for cube in result:
+            self.assertEMOSCoefficientsAlmostEqual(
+                cube.data, self.expected_mean_predictor_each_grid_point[cube.name()],
+            )
+            self.assertIn(cube.name(), self.expected_coeff_names)
+            self.assertEqual(
+                [c.name() for c in cube.coords(dim_coords=True)],
+                ["latitude", "longitude"],
+            )
+
+    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
+    def test_each_point_sites(self):
+        """Test computing coefficients independently for each site location
+        (initial guess and minimising) returns the expected coefficients and
+        associated metadata."""
+        plugin = Plugin(self.distribution, each_point=True)
+        result = plugin.process(self.historic_forecast_spot_cube, self.truth_spot_cube)
+        for cube in result:
+            self.assertEMOSCoefficientsAlmostEqual(
+                cube.data, self.expected_mean_predictor_each_site[cube.name()],
+            )
+            self.assertIn(cube.name(), self.expected_coeff_names)
+            self.assertEqual(
+                [c.name() for c in cube.coords(dim_coords=True)], ["spot_index"],
+            )
+
+    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
+    def test_each_point_sites_realizations(self):
```

Comment:
For completeness, this is equivalent to `test_minimise_each_point_realizations`, if such a test existed.

Reply:
This test `test_minimise_each_point_realizations` now exists and gives a different result to `test_each_point_sites_realizations`.